java/pending_code_changes.txt

```diff
@@ -1 +1,8 @@
+Build changes:
+-- OSGi support added to Manifest information when building jar (#1300)
 
+Code changes:
+-- Formatting, naming (LOGGER -> logger) and comment tweaks to follow style
+guide
+-- Removal of unneeded canBeGeocoded method in the
+PhoneNumberToTimeZonesMapper.java, using phoneNumberUtil instead
```

Comment:
The method removal is for porters, to prompt them to do the same. The style fixes are in case they see tons of stuff changed and want to check it doesn't affect them :)